pkg/config/data/config_schema_v1.0.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "Cog's config YAML schema",
+  "description": "This schema is used to validate cog.yaml file in each project.cog.yaml defines how to build a Docker image and how to run predictions on your model inside that image.",
+  "default": {},
+  "examples": [
+    {
+      "build": {
+        "python_version": "3.8"
+      }
+    }
+  ],
+  "required": [
+    "build"
```

Comment:
I'm not sure build should be required. There are defaults  `python_version`, `cuda`, etc.

Reply:
So build is made "required" to avoid this issue: https://github.com/replicate/cog/issues/34
If someone misspells "build" keyword then cog assumed no config is present and was using default configs. 

Making it required avoids this issue.